unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -46,7 +46,7 @@ export class CreatorLockForm extends React.Component {
       keyPrice: keyPrice,
       keyPriceCurrency: props.keyPriceCurrency,
       maxNumberOfKeys: props.maxNumberOfKeys,
-      unlimitedKeys: props.maxNumberOfKeys === INFINITY,
+      unlimitedKeys: props.maxNumberOfKeys === UNLIMITED_KEYS_COUNT,
```

Comment:
this change should not be made, INFINITY is the text symbol

Reply:
same. I wanted to distinguish between what we consider to be INFINITY (`UNLIMITED_KEYS_COUNT`) and what is actually being displayed `INFINITY`.

The confusion between the 2 is the reason why we had this bug this morning